src/Compilers/CSharp/Portable/Symbols/Source/SourceModuleSymbol.cs

```diff
@@ -619,6 +619,39 @@ internal override void AddSynthesizedAttributes(ModuleCompilationState compilati
             }
         }
 
+        private NamedTypeSymbol _lazyNullableAttribute;
```

Comment:
>private NamedTypeSymbol _lazyNullableAttribute; [](start = 8, length = 47)

Please keep all field declarations at the top of the type declaration.

Reply:
Moved.